.travis.yml

```diff
@@ -10,7 +10,7 @@ env:
   global:
     - DISPLAY=':99.0'
     - YARN_VERSION='0.24.5'
-    - MC_COMMIT='0ef34a9ec4fb' # https://hg.mozilla.org/mozilla-central/shortlog
+    - MC_COMMIT='00ef5405cd7e' # https://hg.mozilla.org/mozilla-central/shortlog
```

Comment:
Checks pass but this is something I'm unfamiliar with; what is the significance of this commit and is there something special I need to test?

Reply:
nope. it changes what travis uses as the latest mc commit